src/test/fuzz/float.cpp

```diff
@@ -19,7 +19,7 @@ FUZZ_TARGET(float)
 
     {
         const double d{[&] {
-            double tmp;
+            double tmp{0};
```

Comment:
The goal of leaving this uninitialized is to have memory sanitizers yell. By initializing you disable memory sanitizers.

Reply:
One workaround would be to make this optional, then `return *Assert(tmp);`. But I am not sure if this is worth it.

Reply:
ah I didn't think about that. I'll remove this commit